src/components/docPage/type.scss

```diff
@@ -68,7 +72,7 @@
   hgroup h2 {
     font-weight: 300;
     font-size: 1.3rem;
-    color: var(--foreground-secondary);
+    color: var(--desatPurple2);
```

Comment:
was there a special reason for this?

Reply:
Yes, it was never done by our design team, and I think the grey that we had before simply was off. Personally, I think it looks much cleaner now

## before
<img width="1149" alt="Screenshot 2024-10-31 at 16 59 02" src="https://github.com/user-attachments/assets/4b8c672a-c6c3-4c64-9a0a-e752dd694fbb">
<img width="1153" alt="Screenshot 2024-10-31 at 16 58 58" src="https://github.com/user-attachments/assets/bb4cbac3-474e-46bd-925e-c092d202641f">


## after

<img width="1127" alt="Screenshot 2024-10-31 at 16 58 44" src="https://github.com/user-attachments/assets/84d7257b-5528-49c9-889f-1756525499ee">
<img width="1143" alt="Screenshot 2024-10-31 at 16 58 39" src="https://github.com/user-attachments/assets/c1fb74cf-826d-48c8-8521-6a8401d52b1b">


what do you think?

Reply:
I think the color in light mode is a bit strong but that's fine 👍